bin/problem.py

```diff
@@ -107,17 +107,9 @@ def _read_settings(self):
         self.settings = argparse.Namespace(**self.settings)
 
         # Override settings by command line arguments.
-        try:
-            self.settings.timelimit = config.args.timelimit or self.settings.timelimit
-        except AttributeError:
-            pass
-
-        timeout = 1.5 * self.settings.timelimit + 1
-        try:
-            if config.args.timeout:
-                timeout = config.args.timeout
-        except AttributeError:
-            pass
+        self.settings.timelimit = config.args.timelimit or self.settings.timelimit
+
+        timeout = config.args.timeout or 1.5 * self.settings.timelimit + 1
         self.settings.timeout = int(timeout)
```

Comment:
you can inline `timeout` here

Reply:
Thanks, I also removed the `int(...)`, because this should all be `int`s already anyway :slightly_smiling_face: (57be5a8)

Reply:
Woops, this turned out to be a `float`, obviously. Time to stop programming for tonight :innocent: (force-pushed to cc6107c, as if nothing ever happened :innocent:)